admin/src/app/shared/constants/utils.ts

```diff
@@ -13,5 +13,15 @@ export default class Utils {
       return re.test(String(email).toLowerCase());
     }
   }
-}
 
+  static singleValidateEmail(email) {
+    var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    return re.test(String(email).toLowerCase());
+  }
+
+  static validatePassword(password) {
+    var re = /^.{6,}$/;
```

Comment:
assign max password length to a variable

Reply:
done